scripts/participateInAuction.js

```diff
@@ -1,19 +1,25 @@
 /* eslint-disable no-console */
 const Coordinator = artifacts.require("Coordinator")
+const { Gastimator } = require("./gas_station")
+
+const GasStation = new Gastimator()
+
+// TODO - make this part of configuration
+const url = "https://safe-relay.rinkeby.gnosis.pm/api/v1/gas-station/"
```

Comment:
Let's do this inside this PR.

Reply:
Yes absolutely. It is already ready. Check it out!